src/AxelarGateway.sol

```diff
@@ -206,6 +206,26 @@ abstract contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
         return getBool(_getIsCommandExecutedKey(commandId));
     }
 
+    /// @dev Returns the current `adminEpoch`.
+    function getAdminEpoch() external view override returns (uint256) {
+        return _adminEpoch();
+    }
+
+    /// @dev Returns the admin threshold for a given `adminEpoch`.
+    function getAdminThreshold(uint256 adminEpoch) external view override returns (uint256) {
```

Comment:
Do we really care about the threshold of a past epoch ever? If not, I'd rather these only being capable returning the latest.